src/Illuminate/Database/Eloquent/Collection.php

```diff
@@ -525,6 +525,28 @@ public function makeVisible($attributes)
         return $this->each->makeVisible($attributes);
     }
 
+    /**
+     * Set the visible attributes across the entire collection.
+     *
+     * @param  array<string>  $visible
```

Comment:
```suggestion
     * @param  array<int, string>  $visible
```

Reply:
Should we also make this change to the underlying method in `HidesAttributes`?

https://github.com/laravel/framework/blob/a615378002db978339493e9af05be9c81cf3187f/src/Illuminate/Database/Eloquent/Concerns/HidesAttributes.php#L54-L60

Reply:
@jessarcher Yes please.